javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -336,8 +337,11 @@ public static Tree getAssignmentContext(final TreePath treePath) {
                 (node instanceof NewArrayTree) ||
                 (node instanceof NewClassTree) ||
                 (node instanceof ReturnTree) ||
-                (node instanceof VariableTree))
+                (node instanceof VariableTree)) {
             return node;
+        } else if (node instanceof ConditionalExpressionTree) {
```

Comment:
It would be good to also update the documentation of the method.
In particular, it says "The assignment context for the treepath is the most enclosing tree of type xyz"; however, the current implementation only returns a Tree if the parent already is one of those Tree kinds. If it is not, it simply returns null at the bottom of the method.
Should the change be to change the last line to "return getAssignmentContext(path);" to always handle the recursive case if it's not one of the leaf cases?